<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-VUHVFPU" name="org.altairretro.practice.CashInfoUpdatetest" tests="1" failures="1" timestamp="2022-09-20T14:22:57 IST" time="4.712" errors="0">
  <testcase name="main" time="4.712" classname="org.altairretro.practice.CashInfoUpdatetest">
    <failure type="java.lang.AssertionError" message="did not expect [I&#039;m MACC JCC
From usa New York] but found [I&#039;m MACC JCC
From usa New York]">
      <![CDATA[java.lang.AssertionError: did not expect [I'm MACC JCC
From usa New York] but found [I'm MACC JCC
From usa New York]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertNotEquals(Assert.java:1399)
at org.testng.Assert.assertNotEquals(Assert.java:1404)
at org.tyss.sales_inventory.Generic.WebDriverUtility.toperformVerificationByAssertNotEquals(WebDriverUtility.java:287)
at org.altairretro.practice.CashInfoUpdatetest.main(CashInfoUpdatetest.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- main -->
  <system-out/>
</testsuite> <!-- org.altairretro.practice.CashInfoUpdatetest -->
